activesupport/lib/active_support/encrypted_configuration.rb

```diff
@@ -11,8 +11,9 @@ class EncryptedConfiguration < EncryptedFile
     delegate :[], :fetch, to: :config
     delegate_missing_to :options
 
-    def initialize(config_path:, key_path:, env_key:)
-      super content_path: config_path, key_path: key_path, env_key: env_key
+    def initialize(config_path:, key_path:, env_key:, raise_if_missing_key:)
+      super content_path: config_path, key_path: key_path,
+        env_key: env_key, raise_if_missing_key: raise_if_missing_key
```

Comment:
Do we have style preferences for kwargs? I like splitting them up to be on newlines:

```ruby
super content_path: config_path, 
      key_path:     key_path,
      env_key:      env_key, 
      raise_if_missing_key: raise_if_missing_key
```

or

```ruby
super content_path: config_path, 
        key_path:   key_path,
        env_key:    env_key, 
        raise_if_missing_key: raise_if_missing_key
```

But it does add a lot of extra whitespace

Reply:
As far as I know, we do not have style preferences for kwargs. But I think the first format you presented is seems good. I updated to use it.